### PR TITLE
fix linking for constants

### PR DESCRIPTION
**Pull request by keertip**

### CODE REVIEW

File: lib/src/model.dart [hunk 1]
```diff
@@ -1028,7 +1028,8 @@ class Field extends ModelElement {
     if (_field.node == null) return null;
     var v = _field.node.toSource();
     if (v == null) return null;
-    return v.substring(v.indexOf('=') + 1, v.length).trim();
+    var string = v.substring(v.indexOf('= ') + 2, v.length);
```
Comment (sethladd) [inline lib/src/model.dart]: Not all constants have a space between the `=`. I think we need to roll this line back.

File: test/fake_package/lib/example.dart [hunk 2]
```diff
@@ -11,6 +11,7 @@ get y => 2;
 
 const String COLOR = 'red';
 const String COMPLEX_COLOR = 'red' + '-' + 'green' + '-' + 'blue';
+const ConstantCat MY_CAT = const ConstantCat('tabby');
```
Comment (sethladd) [inline test/fake_package/lib/example.dart]: Please also add a case for `const String COLOR='green'` and other cases of random spacing.

**Comment by sethladd** [comment]
Disclaimer: just assumptions. testing different scenarios now.

File: lib/src/model.dart (hunk 1 above)
Comment (keertip) [inline lib/src/model.dart]: Will do, and also add a test case.

**Comment by sethladd** [comment]
I added a few more scenarios to the test. Everything looks good!

**Pull request merged**